src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/Baselines/GeneratedAttributesTest.netfx.lang11.g.cs

```diff
@@ -338,47 +338,53 @@ public __SourceGen__RangeAttribute(global::System.Type type, string minimum, str
         public bool ConvertValueInInvariantCulture { get; set; }
```

Comment:
For these properties that have a public setter, who is calling set on them, and when? They're used as part of IsValid, so I'm wondering if it's possible they might be changed concurrently with its execution?

Reply:
This is helper properties usually called only when declaring the RangeAttribute. Something like ` [RangeAttribute(typeof(TimeSpan), "01:00:00", "23:59:59", ConvertValueInInvariantCulture = true)]`.